regulations/static/regulations/js/source/views/comment-review-view.js

```diff
@@ -49,7 +49,7 @@ var CommentReviewView = Backbone.View.extend({
 
   serialize: function() {
     return {
-      docNumber: this.docNumber,
+      doc_number: this.docNumber,
```

Comment:
I don't. I can look into eregs/notice-and-comment#23. However, for the MVP, as it is only a single document, we can get by with just "configuring" it.